AppServer_Java/src/com/google/appengine/tools/resources/ResourceLoader.java

```diff
@@ -7,21 +7,16 @@
     private static ResourceLoader res                       = null;
     private final int             PROTOCOL_BUFFER_PORT      = 8888;
     private final String          DB_LOCATION_PROPERTY      = "DB_LOCATION";
-    private static String         apphome                   = "/root/appscale";
-    private final static String   APPSCALE_HOME_PROPERTY    = "APPSCALE_HOME";
+    private static String         apphome                   = "/etc/appscale";
     private final String          HADOOP_PATH               = "/AppDB/hadoop-0.20.0";
```

Comment:
Since the code using this was prefixing with apphome, this reference is invalid now. It can be removed along with the code that references it.